napari/components/_tests/test_viewer_model.py

```diff
@@ -119,6 +119,15 @@ def test_add_points():
     assert viewer.dims.ndim == 2
 
 
+def test_single_point_dims():
+    """Test dims of a Points layer with a single 3D point."""
+    viewer = ViewerModel()
+    shape = (1, 3)
+    data = np.zeros(shape)
+    viewer.add_points(data)
+    assert all(r == (0.0, 1.0, 1.0) for r in viewer.dims.range)
```

Comment:
@grlee77 naively I would expect [-0.5, 0.5) to be the default range in this scenario, can you explain why it isn't?

Reply:
It will probably allow making this PR simpler to use [-0.5, 0.5) (in general [-step/2, step/2)). I wasn't sure if that was desired or if there was a good reason to keep it closer to how it is in v0.4.11?

In v0.4.11 the ranges in this test would be (0.0, 0.0, 1.0) instead, but then there is a +1 here in `Dims.nsteps` that was since removed:
https://github.com/napari/napari/blob/35d196f66cebb4455b4162685e0ad4519b6bb36d/napari/components/dims.py#L151-L157

I was trying to make this behave as close to that as possible without adding the +1 back to nsteps, but that may not be what we want!